README.rst

```diff
@@ -49,3 +48,29 @@ variable. Then start ``jupyterhub-idle-culler`` manually
 
     export JUPYTERHUB_API_TOKEN=$(jupyterhub token)
     python3 -m jupyterhub-idle-culler[--timeout=900] [--url=http://localhost:8081/hub/api]
+
+The command line interface also gives a quick overview of the different options for configuration.
+
+.. code:: bash
+
+	  --concurrency                    Limit the number of concurrent requests made
```

Comment:
This is great, thank you very much!

So this seems to use a mixture of tabs and spaces. Do you think you can convert it to use just spaces? If not I can do so before merging as well.

Thank you for the PR!

Reply:
I have done so, now it should be clean. Thank you for your great projects that help the world to run JupyterHubs with so much functionality!